app/client/cli/consensus.go

```diff
@@ -0,0 +1,108 @@
+package cli
+
+import (
+	"fmt"
+
+	"github.com/pokt-network/pocket/rpc"
+	"github.com/spf13/cobra"
+)
+
+func init() {
+	consensusCmd := NewConsensusCommand()
+	rootCmd.AddCommand(consensusCmd)
+}
+
+func NewConsensusCommand() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:     "Consensus",
+		Short:   "Consensus specific commands",
+		Aliases: []string{"consensus"},
+		Args:    cobra.ExactArgs(0),
+	}
+
+	cmd.AddCommand(consensusCommands()...)
+
+	return cmd
+}
+
+func consensusCommands() []*cobra.Command {
+	cmds := []*cobra.Command{
+		{
+			Use:     "RoundState",
+			Short:   "Returns \"Height/Round/Step\"",
+			Long:    "RoundState returns the height, round and step in \"Height/Round/Step\" format",
+			Aliases: []string{"round_state"},
+			RunE: func(cmd *cobra.Command, args []string) error {
+				response, err := getConsensusRoundState(cmd)
+				if err != nil {
+					return err
+				}
+
+				fmt.Printf("%d/%d/%d\n", response.JSONDefault.Height, response.JSONDefault.Round, response.JSONDefault.Step)
+
+				return nil
+			},
+		},
+		{
+			Use:     "Height",
+			Short:   "Returns the Height",
+			Long:    "Height returns the height in the consensus state",
```

Comment:
NIT:

```suggestion
			Long:    "Height returns the height in the node's current consensus state",
```

Ditto below.